### ISSUE TITLE
[Object object] error on enabling timed trigger function

### ISSUE THREAD

**Issue by AzureOle** [Azure/Azure-Functions#2377]
New development in a function app has lead to a handful of functions being deployed with a [Disabled] attribute in C#. We're ready to test one of these functions, but we're unable to enable it.

Other functions can be disabled/enabled via the azure portal UI without issue. I recognize that this has been a problem in the past for functions with dashes in their names, but this particular function does not contain a dash. From the function app > Functions overview, attempting to enable fires a generic:

```
"Failed to enable <functionappname>/<myFunctionName>. Error: [Object object]"
```

I've tried via azure CLI as well, 
`
<me>[ ~ ]$ az functionapp config appsettings set --name <myFunctionAppName> --resource-group <myRgName> --settings AzureWebJobs.<myFunctionName>.Disabled=false --output table`

This does not fire an error, and returns 
```

Name                                             SlotSetting    Value
...
AzureWebJobs.<myFunctionName>.Disabled           False          false

```

But the function remains disabled in the Azure UI and doesn't appear to execute. The job fires successfully when run manually.

**Comment by AzureOle** [comment]
This was my mistake over the function of the [Disabled] attribute, I erroneously thought I could then enable post-deployment

**Comment by vijaykumar911** [comment]
Hi @AzureOle thanks for information will update you.

**Comment by vijaykumar911** [comment]
you need to remove the [Disabled] attribute from the function's code and redeploy the function. You mentioned that the function was deployed with the [Disabled] attribute, and you were unable to enable it afterwards. In this case, you should locate the function's code and remove the [Disabled] attribute from the function definition.
Once you have removed the attribute, redeploy the function to your Function App. After deployment, the function should be enabled and ready to execute.

**Comment by AzureOle** [comment]
Yeah, I landed there shortly after opening the issue. Sorry about that. Feel free to close

That said, it would be nice if there was some indicator other than the generic [object Object] on enable, if possible.
